sdks/python/apache_beam/transforms/external.py

```diff
@@ -42,6 +44,63 @@
   grpc = None
 # pylint: enable=wrong-import-order, wrong-import-position, ungrouped-imports
 
+DEFAULT_EXPANSION_SERVICE = 'localhost:8097'
```

Comment:
+1 to moving this here rather than as a default parameter everywhere, though it still doesn't feel right to have it be a global.